br/pkg/lightning/restore/meta_manager.go

```diff
@@ -605,6 +621,74 @@ func (m *dbTaskMetaMgr) CheckClusterSource(ctx context.Context) (int64, error) {
 	return source, nil
 }
 
+// checkEmptyRegion checks if there are too many empty regions on each store.
+func (m *dbTaskMetaMgr) checkEmptyRegion(ctx context.Context) error {
+	var result pdapi.RegionsInfo
+	if err := m.pdTLS.GetJSON(ctx, pdEmptyRegions, &result); err != nil {
+		return errors.Trace(err)
+	}
+	regions := make(map[uint64]int)
+	for _, region := range result.Regions {
+		for _, peer := range region.Peers {
+			regions[peer.StoreId]++
```

Comment:
It seems that we do not need to check whether it has existed?

Reply:
`regions[peer.StoreId]++` is equivalent to `regions[peer.StoreId] = regions[peer.StoreId] + 1`